### PR TITLE
[CC Automation] Add test covering interaction between File screen and published Content Views

### PR DESCRIPTION
**Pull request by sambible**
### Problem Statement
Automation for this BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2026701
Requires airgun PR https://github.com/SatelliteQE/airgun/pull/1426

### Related Issues
This area of Robottelo actually has no tests, so in a separate PR I'm going to raise at least 1 or 2 small tests for the other functionality in this area, just to get some coverage. 

trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_file_cv_display'
airgun: 1426

### CODE REVIEW

**Comment by LadislavVasina1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_file_cv_display'
airgun: 1426

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7400
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_file_cv_display --external-logging
Test Result : ============ 3 deselected, 9 warnings, 1 error in 198.70s (0:03:18) ============
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7412
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_file_cv_display --external-logging
Test Result : =========== 1 passed, 3 deselected, 14 warnings in 662.09s (0:11:02) ===========
```

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+
+
+@pytest.mark.tier2
+def test_file_cv_display(session, target_sat, module_org, module_product):
+    """Content-> Files displays only the Content Views associated with that file
+
+    :id: 41719f2f-2170-4b26-b65f-2063a1eac7fb
+
+    :steps:
+        1. Create a File Repo, and publish content to it
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:120]:
```suggestion
        1. Create a file repo, and upload content into it
```

File: tests/foreman/ui/test_contentview.py [hunk 2]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+        2. Add it to a CV, and publish that CV
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:121]:
```suggestion
        2. Add file repo to a CV, and publish it
```

File: tests/foreman/ui/test_contentview.py [hunk 3]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+        3. Create a seperate CV, and publish it as well.
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:122]:
```suggestion
        3. Create another CV, and publish it
```

File: tests/foreman/ui/test_contentview.py [hunk 4]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+        4. Navigate to the Content -> File section of the UI
+
+    :expectedresults: The CV containing the file repo is displayed, and no other CVs.
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:125]:
```suggestion
    :expectedresults: Only the Content View with the file repo is displayed.
```

File: tests/foreman/ui/test_contentview.py [hunk 5]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+
+    :BZ: 2026701
+
+    :customerscenario: true
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:129]: Can we add Verifies field as well?
```suggestion
    :customerscenario: true
    
    :Verifies: SAT-17081
```

Make sure the Jira id `SAT-17081` is correct  :smile_cat:

File: tests/foreman/ui/test_contentview.py [hunk 6]
```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
...
+    """
+    repo_name = gen_string('alpha')
+    file_repo = target_sat.api.Repository(
+        product=module_product, name=f'{repo_name}_file_repo', content_type='file'
+    ).create()
+    with open(DataFile.FAKE_FILE_NEW_NAME, 'rb') as handle:
+        file_repo.upload_content(files={'content': handle})
+    assert file_repo.read().content_counts['file'] == 1
+    cv = target_sat.api.ContentView(organization=module_org).create()
+    cv = target_sat.api.ContentView(id=cv.id, repository=[file_repo]).update(["repository"])
```
Comment (vijaysawant) [inline tests/foreman/ui/test_contentview.py:139]:
```suggestion
    cv = target_sat.api.ContentView(id=cv.id, repository=[file_repo]).update(['repository'])
```

**Review by vijaysawant — CHANGES_REQUESTED** [review]
- Trying to correct sentances (I might be wrong) from test doc string > steps & expected results
- Suggestion to use `Verifies` field
- Suggestion to use single quotes for strings using inside test module to make it consistent.

**Review by vijaysawant — APPROVED** [review]
Thank you for work, looks good to me.
Pending PRT result.

**Comment by vijaysawant** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_file_cv_display'
airgun: 1426

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7440
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_file_cv_display --external-logging
Test Result : =========== 1 passed, 3 deselected, 14 warnings in 681.55s (0:11:21) ===========
```

**Review by vsedmik — APPROVED** [review]

**Review by damoore044 — APPROVED** [review]

**Pull request merged**